pynest/examples/clopath_synapse_spike_pairing.py

```diff
@@ -115,17 +115,16 @@
     prrt_nrn = nest.Create("parrot_neuron", 1)
 
     # Create and connect spike generators
-    spike_gen_pre = nest.Create("spike_generator", 1, {
-                                "spike_times": s_t_pre})
+    spike_gen_pre = nest.Create("spike_generator", 1,
+                                {"spike_times": s_t_pre})
 
     nest.Connect(spike_gen_pre, prrt_nrn,
                  syn_spec={"delay": resolution})
 
-    spike_gen_post = nest.Create("spike_generator", 1, {
-                                 "spike_times": s_t_post})
+    spike_gen_post = nest.Create("spike_generator", 1,
+                                 {"spike_times": s_t_post})
```

Comment:
In the examples above, the `1` is removed when 1 node is created. Maybe remove all here as well for consistency over the examples?

Reply:
agree